keyvalue/clients/src/main/java/tachyon/client/keyvalue/hadoop/KeyValueOutputCommitter.java

```diff
@@ -34,86 +41,64 @@
  * Extension of {@link FileOutputCommitter} where creating, completing, or deleting a
  * {@link KeyValueStores} in different phases of a job's or task's lifecycle is considered.
  * <p>
- * This committer is forced to be used in {@link KeyValueOutputFormat} (no matter what users have
- * set as the {@link org.apache.hadoop.mapred.OutputCommitter} in configuration) to merge the
- * key-value stores created by each Reducer into one key-value store under the MapReduce output
- * directory.
+ * This committer must be used along with {@link KeyValueOutputFormat} to merge the key-value stores
+ * created by each Reducer into one key-value store under the MapReduce output directory.
  */
+@PublicApi
```

Comment:
It's coupled with changes to output format, I combined them in a single commit: https://github.com/uronce-cc/tachyon/commit/23f53c545b4b2e51f2473f99d3835f4078a80acc